site/app/templates/autograding/TAResults.twig

```diff
@@ -28,17 +28,39 @@
         </div>
         {% if any_late_days_used %}
             <a name="show_late_table" style="float:right" class="btn btn-primary" href="{{ late_days_url }}">Show my late days information</a>
+            <br />
+            <br />
         {% endif %}
         {% if uploaded_pdfs|length >= 1 %}
-        <div class="box half" style="float: right; width: 25%; word-break: break-word">
-            View/Download submission with feedback:
-            <br />
-            {% for file in uploaded_pdfs %}
-                {{ file.name }}
-                {#<a class="btn btn-primary" onclick="openUrl('{{ core.buildUrl({'component':'student', 'page':'PDF', 'action':'download_annotated_pdf', 'gradeable_id': gradeable_id, 'file_name': file.name}) }}')">Download <i class="fas fa-download"></i></a>#}
-                <a class="btn btn-primary" onclick="openUrl('{{ core.buildUrl({'component':'pdf', 'page':'student_pdf_view', 'gradeable_id': gradeable_id, 'file_name': file.name}) }}')">View Popup <i class="fas fa-window-restore"></i></a>
-            {% endfor %}
-        </div>
+
+            <div class="box half" style="float: right; width: 25%; word-break: break-word">
+                <p>View/Download submission with feedback:</p>
+                {% for file in uploaded_pdfs %}
+                    <p> {{ file.name }}
+                    {#<a class="btn btn-primary" onclick="openUrl('{{ core.buildUrl({'component':'student', 'page':'PDF', 'action':'download_annotated_pdf', 'gradeable_id': gradeable_id, 'file_name': file.name}) }}')">Download <i class="fas fa-download"></i></a>#}
```

Comment:
Can this comment be removed?

Reply:
This is there for when annotation download becomes available, which currently is not working because the capability is not there from the pdf provider.